gammapy/maps/regionnd.py

```diff
@@ -189,6 +189,27 @@ def create(cls, region, axes=None, dtype="float32", meta=None, unit="", wcs=None
     def downsample(
         self, factor, preserve_counts=True, axis_name="energy", weights=None
     ):
+        """Downsample the non-spatial dimension by a given factor.
```

Comment:
Those docstrings are defined in the base class as well, but I think here the behaviour slightly differs in the way that the default axis to downsample is "energy", while for a `WcsNDMap` it's the spatial axes. So definitely keep the docstring here!